### PR TITLE
Fixes #829 - README Translation to PT-BR Done

### LINKED ISSUE #829: PT-BR translation for README.md
I'm thinking in writing a translation for the readme file in pt-br. I know a lot of friends here in Brazil that uses this lib and actually can't speak/read english. What do you guys think about it and how should we make that? (I'm willing to translate)

### PR DESCRIPTION
**Pull request by gustavomfb**
## Fixes

Fixes #829 by @gustavomfb

## Description
Full translation of README.md file to PT-BR, action button on first HTML tag (PT-BR Readme)
On README.pt.md there's a button that goes to EN version too.

## Checklist
[x] My pull request has a descriptive title (not a vague title like Update index.md).
[x] My pull request targets the default branch of the repository (main) or a parent feature branch.
[x] My commit messages follow [best practices](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines).
[x] My code follows the established code style of the repository.
[] I added or updated tests for the changes I made (if applicable).
[x] I added or updated documentation (if applicable).
[x] I tried running the project locally and verified that there are no visible errors.

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by gustavomfb** [comment]
Let me know when u guys are able to review my PR!

**Comment by krysal** [comment]
I replaced my requirement of review with Sara's as @sarayourfriend is way well versed in Portuguese than me :)

File: README.pt.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+└── *                                       # Documentação, arquivos de configuração, requerimentos de projeto
+```
+
+## Publicando
+
+A imagem do docker para o catalogo (Airflow) é publicada para o ghcr.io/WordPress/openverse-catalog.
+
+## Contribuindo
+
+Pull requests são bem-vindos! Sinta-se a vontade para nos juntar no [join us on Slack][wp_slack] e discutir o projeto com os engenheiros e membros da comunidade no #openverse
```
Comment (sarayourfriend) [inline README.pt.md:235]:
```suggestion
Pull requests são bem-vindos! Sinta-se a vontade para nos juntar no [Slack do WordPress Make][wp_slack] e discutir o projeto com os engenheiros e membros da comunidade no #openverse.
```

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
Muito obrigada pelo PR! I made some suggestions for improvements/typo corrections. Overall it looks great to me. The second reviewer aside from me will just have to rubber stamp this and trust us on it, I'm not aware of any other contributors that know Portuguese :slightly_smiling_face: Bemvindo o projecto!

File: README.pt.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+de cerca de 200 TiB de dados descompactados.
+
+Os dados estão disponiveis em três formatos de arquivo:
+
+- WARC (Web ARChive): Toda a informação crua, incluindo metadata de respostas HTTP,
+  metadata de WARC, etc.
+- WET: texto sem formatação de cada página.
+- WAT: extração de metadata de html, e.g. cabeçalhos e hyperlinks HTTP, etc.
+
+Para mais informações a cerca destes formatos, por favor veja:
```
[the remark this answers is not in the document]
Comment (gustavomfb) [reply README.pt.md:31]: A cerca é um formato mais formal, geralmente usado em textos de pesquisa/projetos que também remete ao "sobre", entretanto, é gramaticalmente mais reconhecido como "a respeito"!

File: README.pt.md (hunk 1 above)
Comment (gustavomfb) [reply README.pt.md]: Sara, ja inclui todas as modificações sugeridas. Em alguns outros lugares coloquei algo que coloquialmente estava mais correto! Agradeço desde já!

File: README.pt.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sarayourfriend) [reply README.pt.md:31]: Ahhh, entendi! Obrigada pela explicação :pray:

[the remark this answers is not in the document]
Comment (gustavomfb) [reply README.pt.md:31]: > Ahhh, entendi! Obrigada pela explicação 🙏

ainda há alguma coisa que deve ser arrumada?

[the remark this answers is not in the document]
Comment (gubenites) [reply README.pt.md:31]: @satahyourfriend

[the remark this answers is not in the document]
Comment (gubenites) [reply README.pt.md:31]: @sarayourfriend

[the remark this answers is not in the document]
Comment (sarayourfriend) [reply README.pt.md:31]: Desculpa, eu estava de férias. Os maintainers tem umas dúvidas se é mesmo uma boa ideia tentar manter traduções do README. Por enquanto estou achando que talvez seria muito trabalho tentar manter os READMEs em vários idiomas... Vocês já viram que o README em inglês mesmo tem problemas de não estar completamente atualizada com as condições do projeto.

O que que vocês acham sobre isso? Vale muito a pena ter traduções do README para justificar o trabalho de manter-lhes?

**Comment by sarayourfriend** [comment]
I'm going to close this PR for now. This discussion still remains unresolved and the team is sceptical to commit this given the clear possibility of drift and difficulty in maintaining even one translation. Given this would set a precedent for adding additional translations, it seems even riskier.

https://github.com/WordPress/openverse-catalog/pull/838#discussion_r1011007564

One alternative that would be worth exploring, I think, is for us to submit the generated catalogue and API documentation for translation the same way we do for the Openverse Frontend. We'd need to spend considerable time deciding how the infrastructure for that would work, specific how we would deploy multiple translations. For the API, Django readily supports gettext (by way of ugettext) and PO files. I wonder if Sphinx has something in this vein as well.

@gustavomfb If you'd like to work on that, please let me know, and we can open some issues for it. Otherwise, thank you for this contribution, but it is not the right time for us to invest in this and set a precedent towards it that we're not able to commit to maintaining.

**Pull request closed without merging**